site/profile/templates/cvmfs/config.d.epp

```diff
@@ -0,0 +1,7 @@
+<%# Create repo to store alien cache instead of having them created/store in default.local file
+<% if $alien_cache_repositories != '' { -%>
+CVMFS_REPOSITORIES='<%= $alien_cache_repositories.join(',') %>'
+CVMFS_ALIEN_CACHE='/scratch/cvmfs_alien_cache/@fqrn@'
```

Comment:
- Who is responsible of creating `/scratch/cvmfs_alien_cache`? CVMFS or Puppet?
- Who will own that directory?

Scratch is an NFS mounted directory with no-root squash. If the directory is created before NFS is mounted, the alien cache will not be visible. If puppet tries to create the directory after NFS scratch is mounted, it will fail because of the no-root squash.

Is the alien cache supposed to be on NFS? If so, the folder should be created on the NFS server, not the CVMFS clients.

If the alien cache should be local to every CVMFS client, it should be created in its own local folder different than `/scratch`.

Reply:
Done, I define it under `cvmfs_cache` instead -> new name